README.md

```diff
@@ -88,3 +88,45 @@ $authorizationChecker = $container->get(/* ... */);
 $authorizationChecker->isGranted(array('IS_OWNER'), $germinal); // true
 $authorizationChecker->isGranted(array('IS_OWNER'), $miserables); // false
 ```
+
+##Security from routing
+
+You can specify security constraints directly from your routing.
+
+**Example**
+```yaml
+acme_demo:
+    path: /demo
+    defaults:
+        _controller: FrameworkBundle:Template:template
+        template: Acme:demo:index.html.twig
+        _security:
+            - role: IS_AUTHENTICATED_ANONYMOUSLY
+```
+
+Also, it integrates well with the [rad-resource-resolver](https://github.com/KnpLabs/rad-resource-resolver) component & the ParamConverter from SensioLabs.
+
+**Example**
+```yaml
+acme_group_update:
+    path: /team/{tid}/group/{gid}/update
+    defaults:
+        _controller: AcmeBundle:Group:update
+        template: Acme:Group:update.html.twig
+        _resources:
+            team:
+                service: acme.entity.team_repository
+                method: find
+                arguments: [$tid]
+            group:
+                service: acme.entity.group_repository
+                method: find
+                arguments: [$gid]
+        _security:
+            -
+                role: [IS_MEMBER, ANOTHER_ROLE]
```

Comment:
Will this allow the user to have access if they have both roles, or only one? I think we should add this in the doc, cause it's not intuitive.